move/axelar_gateway/sources/gateway.move

```diff
@@ -176,8 +179,8 @@ entry fun approve_messages(
     message_data: vector<u8>,
     proof_data: vector<u8>,
 ) {
-    let messages = peel_messages(*&message_data);
```

Comment:
`*&message_data` lol

Reply:
this made me question a lot if there's any black magic that i don't know; I just trust the compiler 😂